### ISSUE TITLE
Unable to locate grpc-server node

### ISSUE THREAD

**Issue by XSehio** [julien-pal/node-red-contrib-grpc#25]
I just instaled the package but I can't see the grpc-server node in the node palette. Any idea of why is this happening?
Thanks

**Comment by namgk** [comment]
Any logs from node red?

**Comment by XSehio** [comment]
No! At least that's what I think. Where can I find the logs files?

**Comment by namgk** [comment]
When you start node red, you’ll see the logs. How do you start node red?

**Comment by XSehio** [comment]
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:70.85pt 3.0cm 70.85pt 3.0cm;}
div.WordSection1
	{page:WordSection1;}
